### PR TITLE
feat: create env buttons

### PR DESCRIPTION
**Pull request by neurosnap**
Screenshot 2023-08-16 at 1 43 17 PM
Screenshot 2023-08-16 at 1 43 12 PM

### CODE REVIEW

**Comment by joeclayallday** [comment]
Small tweak to align the New buttons with search bar & tweak the + symbol to match others

Screen Shot 2023-08-16 at 12 35 00 PM

File: src/ui/shared/environment/environment-list.tsx [hunk 1]
```diff
@@ -131,63 +135,53 @@ const EnvironmentListRow = ({ environment }: EnvironmentCellProps) => {
...
+      filterBar={
+        <div>
+          <div className="flex justify-between items-center">
+            <InputSearch
+              placeholder="Search environments..."
+              search={search}
+              onChange={onChange}
+            />
+
+            <ButtonIcon icon={<IconPlusCircle />} onClick={onCreate}>
```
Comment (joeclayallday) [inline src/ui/shared/environment/environment-list.tsx:165]: This is missing `variant="sm"` @neurosnap

**Comment by joeclayallday** [comment]
Small design tweaks
• Made save button fit width of words
• Bolded important text in radio button paragraphs
• Made the radio button label text bigger
Screen Shot 2023-08-17 at 10 03 54 AM

File: src/ui/pages/create-env.tsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+            </FormGroup>
+          ) : (
+            <Banner variant="info">
+              Dedicated Stacks are only available on Scale and Enterprise plans.
+            </Banner>
+          )}
+
+          <hr />
+
+          <Button
```
Comment (madhuravius) [inline src/ui/pages/create-env.tsx:204]: one place the create can fail is if another environment has the same name i think, so as usual probably wanna add the banner alerts (or maybe i missed it)

**Review by madhuravius — COMMENTED** [review]
LGTM, again might wanna include the banner messages

merge conflict / rebase might make it easy
you might also want to include a happy path test on this as i think a lot of users go into this page (i didn't see it on pr, maybe i missed it)

**Review by madhuravius — APPROVED** [review]

**Pull request merged**